src/components/CohortsPreview/StudentSummary.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from 'react';
+import client from '../../utils/client';
+import './cohorts-preview.css'
+
+export default function StudentSummary({ id, handleClick }) {
+  const [studentData, setStudentData] = useState(null);
+
+  useEffect(() => {
+    client.get(`/user/${id}`)
+      .then(res => setStudentData(res.data.data.user))
```

Comment:
Where are you getting ```data.data.user``` from?

Reply:
From the response we get from the GET request on line 9